consensus/src/sync/history/sync_clustering.rs

```diff
@@ -48,13 +48,10 @@ impl<TNetwork: Network> HistoryMacroSync<TNetwork> {
             (locators, election_head.epoch_number())
         };
 
-        let result = Self::request_macro_chain(
-            Arc::clone(&network),
-            peer_id,
-            locators,
-            1000, // TODO: Use other value
-        )
-        .await;
+        let max_epochs = 1000; // TODO: Use other value
```

Comment:
Could be a `const` on the type. 

As it is used by both implementors of the `MacroSync` trait it could also be moved there to be declared by every implementor if that generalization is plausible.